bindings/python/examples/how_tos/client/get_outputs.py

```diff
@@ -18,8 +19,9 @@
 
 # Get output ids of basic outputs that can be controlled by this address without further unlock constraints
 output_ids_response = client.basic_output_ids(query_parameters)
-print(f'{output_ids_response}')
+print('First output of query:')
```

Comment:
Maybe this?
```suggestion
print('First output id of the query response:')
```

Reply:
Hm in the output it makes sense since below the output is also printed (if it doesn't fail), but above this line it looks a bit wrong

Reply:
That's why i moved this print above the other too, so i can show both the first ID and the output itself. But i can change it. Just tell me how 😅

Reply:
Just remove :trollface:

Reply:
Can stay like it is